### PR TITLE
doc: User VM guide

### PR DESCRIPTION
**Pull request by dbkinder**
Create a new User VM OS guide with overview information about setting up
User VMs.  Link to separate standard and RTVM guide for details. Most of
these separate guides need updating to reflect the new ACRN configurator and
the development system/target system development model.

Update titles of existing "using XXX as user vm" to be consistent.

Begin update of ubuntu uservm doc for direction review.

Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Review by NanlinXie — APPROVED** [review]
Thanks @dbkinder , it is a very good update and more clear to user.
We could merge and publish firstly and file new PR if need any update.

File: doc/tutorials/user_vm_guide.rst [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+   Set VM image using virtio block device
+
+After the configuration settings are to your liking, save the configuration.
+When saving, the configurator first validates your scenario configuration and
+reports any issues that need your attention.  If successful, it writes out the
+updated scenario XML file and launch script to your working directory.  You'll
+use this launch script to start the User VM on the target.
+
+Rebuild the ACRN hypervisor
```
Comment (amyreye) [inline doc/tutorials/user_vm_guide.rst:118]:
```suggestion
Rebuild the ACRN Hypervisor
```

File: doc/tutorials/using_ubuntu_as_user_vm.rst [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+#. The KVM image is created in the ``/var/lib/libvirt/images`` folder.
+   Convert the ``gcow2`` format to ``img`` **as the root user**:
+
+   .. code-block:: none
+
+      $ cd ~ && mkdir ubuntu_images && cd ubuntu_images
+      $ sudo qemu-img convert -f qcow2 -O raw /var/lib/libvirt/images/ubuntu20.04.qcow2 ubuntu_uservm.img
+
+*DBK stopped here*
```
Comment (amyreye) [inline doc/tutorials/using_ubuntu_as_user_vm.rst:135]: Delete this note?

File: doc/tutorials/using_ubuntu_as_user_vm.rst [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+      Ubuntu 20.04 LTS and then click **Forward**.
+
+      .. figure:: images/ubuntu_uservm_03.png
+         :align: center
+         :name: newVM-ubun-image
+         :class: drop-shadow
+
+         Select Ubuntu ISO file previously downloaded
+
+   #. Choose **Enable storage** and **Create a disk image for the virtual machine**. Set the
```
Comment (gvancuts) [inline doc/tutorials/using_ubuntu_as_user_vm.rst:88]: Should you not break this line?

File: doc/tutorials/using_ubuntu_as_user_vm.rst [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+      $ cd ~ && mkdir ubuntu_images && cd ubuntu_images
+      $ sudo qemu-img convert -f qcow2 -O raw /var/lib/libvirt/images/ubuntu20.04.qcow2 ubuntu_uservm.img
+
+*DBK stopped here*
+
+Launch the Ubuntu Image as the User VM
+**************************************
+
+Modify the ``launch_win.sh`` script in order to launch Ubuntu as the User VM.
```
Comment (gvancuts) [inline doc/tutorials/using_ubuntu_as_user_vm.rst:140]: We need to use the launch script that was generated by the tool here.

File: doc/tutorials/using_ubuntu_as_user_vm.rst [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+
+.. note:: This tutorial assumes SATA is the default boot drive; replace
+   ``/dev/sda1`` mentioned below with ``/dev/nvme0n1p1`` if you are
+   using an SSD.
```
Comment (gvancuts) [inline doc/tutorials/using_ubuntu_as_user_vm.rst:144]:
```suggestion
   using an NVMe storage device.
```

File: doc/tutorials/using_ubuntu_as_user_vm.rst [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+
+1. Copy the ``ubuntu_uservm.img`` to your target system:
+
+   .. code-block:: none
+
+      # scp ~/ubuntu_images/ubuntu_uservm.img user_name@ip_address:~/ubuntu_uservm.img
+
+#. Log in to the ACRN Service VM, and create a launch script from the existing script:
```
Comment (gvancuts) [inline doc/tutorials/using_ubuntu_as_user_vm.rst:152]: We need to drop this and re-use the launch script from the ACRN Configurator

**Pull request merged**